cmd/root.go

```diff
@@ -34,14 +34,6 @@ var RootCmd = &cobra.Command{
 	Aliases:           []string{"clab"},
 }
 
-// Execute adds all child commands to the root command and sets flags appropriately.
```

Comment:
appears this part was dead code after the (I think) relatively recent move to use fang for the cli